FirebaseMessaging/Sources/FIRMessagingAnalytics.m

```diff
@@ -212,11 +216,34 @@ + (void)logMessage:(NSDictionary *)notification
       break;
 
     default:
-      // Only a silent notification (i.e. 'content-available' is true) can be received while the app
-      // is in the background. These messages aren't loggable anyway.
+      // App was either in background state or in transition from closed
```

Comment:
Would it be possible to add tests for the new logic to guard against regressions in the future?